composer.json

```diff
@@ -9,6 +9,7 @@
     "license": "MIT",
     "require": {
         "php": "^7.2",
+        "doctrine/dbal": "^2.10",
```

Comment:
where do you make use of this package?

Reply:
I said in one of the reply comments ^^, because Laravel wouldn't let me migrate without it. Don't know why, everyone said install it.

Reply:
oh ok, fair enough

Reply:
I can't approve my own PR but consider your changes approved